### DISCUSSION TITLE
Why VTK cannot be found after installing successfully?

### DISCUSSION THREAD

**Discussion by mjjackey** [microsoft/vcpkg#20040] [Q&A] [answered]
I used vcpkg to install vtk, the installation was successful.
![vtk_vcpkg](https://user-images.githubusercontent.com/14541043/132456261-7e07d688-3a1e-470f-a219-36975f199bd0.png)

After integration configuration in visual studio 2019 on Windows 10, the project still cannot find VTK.

![image](https://user-images.githubusercontent.com/14541043/132456362-e1954a6d-78a6-4790-854b-b856545a38b8.png)

What's the reason for this problem?

**Comment by Neumann-A** [comment, marked as answer]
> What's the reason for this problem?

You are not using CMake. There are ports in vcpkg witch the Visual Studio integration can not correctly capture since it is too simple (it just adds `<vcpkg_installed>/include` to the include dirs which is not enough for a port like vtk)

**Reply by mjjackey** [reply]
Thank you very much for your reply! I have used CMake to compile the project. The "VTK cannot be found" problem has been resolved, but does not run to the correct result. There is a warning. I do not know why. (I'm CMake newer.)
![image](https://user-images.githubusercontent.com/14541043/132490186-33949773-9e80-4955-8634-e95677fdeda8.png)

**Reply by Neumann-A** [reply]
https://github.com/trlsmax/imgui-vtk/issues/2
Helps probably
